examples/dynamics/controlled_integrator.ipynb

```diff
@@ -232,7 +233,8 @@
       "plt.plot(t, ref, 'k--', label='Exact')\n",
       "plt.plot(t, sim.data[A_probe][:,0], label='A (value)')\n",
       "plt.plot(t, sim.data[A_probe][:,1], label='A (control)')\n",
-      "plt.ylim([-1.1, 1.1])\n",
+      "plt.xlim(right=t[-1])\n",
```

Comment:
Looking in more detail, it looks like that changed over two years ago, at least. They just never changed the `xlim` docs. So I think `right` is actually the right choice...